src/tiled/invertYCoordinateHelper.h

```diff
@@ -0,0 +1,19 @@
+#ifndef INVERTYCOORDINATEHELPER_H
+#define INVERTYCOORDINATEHELPER_H
+
+#include "preferences.h"                // To get the status of the invertYCoordinate checkBox
+#include "mapdocument.h"                // To access the map
+#include "mapdocumentactionhandler.h"
```

Comment:
None of these includes are needed in the header file. Please move them to the source file to reduce include dependencies.